example/AeroGearSdkExample/ViewController.swift

```diff
@@ -10,15 +10,35 @@ class ViewController: UIViewController, UIPickerViewDataSource, UIPickerViewDele
 
     @IBOutlet var pickerView: UIPickerView!
 
+    @IBOutlet weak var responseLabel: UILabel!
+    @IBOutlet var requestButton: UIButton!
     @IBOutlet var config: UILabel!
 
-    var pickerDataSource = ["sync-server", "prometheus"]
+    var pickerDataSource = ["sync-server", "prometheus", "echo"]
+    
+    @IBAction func buttonClick(sender _: UIButton) {
+        let row = pickerView.selectedRow(inComponent: 0)
+        let http = AgsCore.instance.getHttp(pickerDataSource[row])
+        if let http = http {
+            http.request(method: .post, path: "", completionHandler: { (response, error) -> Void in
```

Comment:
merge into one if let

Reply:
Not sure what statement you mean and I want to avoid misunderstandings. 
What statement you mean?